main.js

```diff
@@ -381,4 +485,15 @@ $cartView.addEventListener('click', (event) => {
     app.view = 'catalog'
     renderAppState(app)
   }
+  else if (event.target.id === 'btn-checkout') {
+    app.view = 'checkout'
+    renderAppState(app)
+  }
+})
+
+var $checkoutView = document.querySelector("[data-view='checkout']")
+$checkoutView.addEventListener('click', (event) => {
```

Comment:
In general, we do not listen for `click` events when dealing with forms. Instead, handle a `submit` event. But be sure to prevent the default behavior of form submissions.